### PR TITLE
docs: update migration

### PR DESCRIPTION
**Pull request by janniks**

### CODE REVIEW

File: .github/MIGRATION.md [hunk 1]
```diff
@@ -188,7 +188,10 @@ For `bigint` values, the type of the `value` property is a now `string`, for bet
 The old `PostCondition` type was renamed to `PostConditionWire`.
 A new human-readable `PostCondition` type was introduced in its place.
 
-Below is an example of the new `PostCondition` types.
+The previous builders (`makeStandardSTXPostCondition`, `makeStandardFungiblePostCondition`, `makeStandardNonFungiblePostCondition`, `makeContractSTXPostCondition`, `makeContractFungiblePostCondition`, and `makeContractNonFungiblePostCondition` using the enums where removed.
```
Comment (ryanwaits) [inline .github/MIGRATION.md]:
```suggestion
The previous builders (`makeStandardSTXPostCondition`, `makeStandardFungiblePostCondition`, `makeStandardNonFungiblePostCondition`, `makeContractSTXPostCondition`, `makeContractFungiblePostCondition`, and `makeContractNonFungiblePostCondition` using the enums were removed.
```

File: .github/MIGRATION.md [hunk 2]
```diff
@@ -188,7 +188,10 @@ For `bigint` values, the type of the `value` property is a now `string`, for bet
...
+Use the `Pc` post-condition builder instead or construct post-condition object (of the type `PostCondition` manually using strings and string literals.
```
Comment (ryanwaits) [inline .github/MIGRATION.md]:
```suggestion
Use the `Pc` post-condition builder instead or construct post-condition object (of the type `PostCondition`) manually using strings and string literals.
```

**Review by ryanwaits — CHANGES_REQUESTED** [review]
just a few typos but lgtm - ill make a new issue to update docs to remove the deprecated helper functions

**Comment by janniks** [comment]
Thx @ryanwaits fixed, can be rereviewed

**Review by ryanwaits — APPROVED** [review]

**Pull request merged**